carma_wm_ctrl/launch/carma_wm_broadcaster.launch

```diff
@@ -15,6 +15,7 @@
   <arg name = "max_lane_width"  default = "4" doc= "Max lane width in meters within which geofence points are associated to a lanelet as those points are guaranteed to apply to a single lane"/>
   <node name="carma_wm_broadcaster" pkg="carma_wm_ctrl" type="carma_wm_ctrl_node">
     <remap from="georeference" to="$(optenv CARMA_LOCZ_NS)/map_param_loader/georeference"/>
+    <remap from="/environment/geofence" to="incoming_geofence_control"/>
```

Comment:
incoming_geofence_control needs $(optenv CARMA_MSG_NS)/ 
infront of it since that topic is from a node launched under a different namespace.